lint-tasks.yml

```diff
@@ -30,6 +53,20 @@ tasks:
           lint-tasks.yml \
           Taskfile.yml
 
+  clang-format:
```

Comment:
Just curious, is there a future plan to have this task moved into yscope-dev-utils and also have the venv setup task done from there?

Reply:
I hadn't considered it, but yeah, it probably makes sense, especially now that we can specify maps/arrays as arguments to a task.